uw-research-computing/gpu-jobs.md

```diff
@@ -169,39 +90,39 @@ benefit from this investment. **Specifically, jobs running on GPU Lab servers ha
 There are a certain number of slots in the GPU Lab reserved for interactive use. Interactive 
 jobs that use GPU Lab servers are restricted to using a single GPU and a 4 hour runtime. 
 
-These job types, runtimes, and per-user limitations are subject to change with
-short notice as the CHTC GPU Lab studies usage patterns.
+## 2. Other Capacity
 
-By opting-in to use the CHTC GPU Lab servers, you agree to be contacted by the
```

Comment:
Is permission to be contacted already implied by using CHTC resources?

Reply:
yep.